adafruit_sht31d.py

```diff
@@ -210,7 +217,7 @@ def _periodic(self):
                 time.sleep(0.001)
                 self._last_read = 0
 
-    def _data(self):
+    def _data(self) -> Tuple[float, float]:
```

Comment:
It looks like this also has the possibility to return a `Tuple[List[float], List[float]]`, so add that inside of the one you've already got inside a `Union`.